src/pages/docs/chat/rooms/messages.mdx

```diff
@@ -7,6 +7,8 @@ Send, update, delete, and receive messages in a chat room with any number of par
 
 A user can also update or delete a message, all users that are subscribed to the room will be notified of the changes.
 
+**Note:** To support updates, deletions, and [reactions](/docs/chat/rooms/message-reactions), [message persistence](/docs/chat/rooms#persistence) is enabled by default for all chat rooms.
```

Comment:
We have an `Aside` block type we can use here (which puts it in a nice callout box)